### PR TITLE
Readded the plainbox examples suite to travis config.

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

**Comment by elopio** [comment]
Do not merge yet. Just looking how it goes.

**Comment by ted-gould** [comment]
I think that merge #63 should fix the QML plugin on Travis.

**Comment by sergiusens** [comment]
Can you disable the qml example testing for now?

File: .travis.yml [hunk 1]
```diff
@@ -15,14 +15,17 @@ env:
...
 install:
   # PPA for plainbox.
   - sudo apt-add-repository -y ppa:hardware-certification/public
   # PPA for snappy.
   - sudo apt-add-repository -y ppa:snappy-dev/tools-proposed
+  # PPA for qtmir.
+  - sudo apt-add-repository -y ppa:ubuntu-sdk-team/testing
   - sudo apt-get update -qq
-  - sudo apt-get install -qq build-essential dpkg-dev plainbox pyflakes python3-apt python3-coverage python3-fixtures python3-jsonschema python3-mccabe python3-pep8 python3-requests python3-testscenarios python3-yaml ubuntu-snappy-cli
+  - sudo apt-get install build-essential dpkg-dev plainbox pyflakes python3-apt python3-coverage python3-fixtures python3-jsonschema python3-mccabe python3-pep8 python3-requests python3-testscenarios python3-yaml qtdeclarative5-qtmir-plugin ubuntu-snappy-cli
```
Comment (sergiusens) [inline .travis.yml]: why do we need qtmir here?

Comment (elopio) [inline .travis.yml]: I was trying to satisfy the requirements. Didn't go anywhere, so I'll rollback this changes and just skip the qml example in vivid.

File: integration-tests/units/examples.pxu [hunk 2]
```diff
@@ -47,7 +47,7 @@ id: snapcraft/examples/py2project
 plugin: shell
 command:
   set -e
-  example=${PLAINBOX_PROVIDER_DATA}/examples/py2project
+  example=${PLAINBOX_PROVIDER_DATA}/examples/py2-project
```
Comment (elopio) [inline integration-tests/units/examples.pxu]: meh, bad review. I should have looked with more attention.

**Comment by elopio** [comment]
Three tests failing here:
godd: /usr/bin/ld: cannot find -lgudev-1.0
It seems go is not looking for libraries on the snap dir.
catkin: Unable to find "package.xml" for "beginner_tutorials"
libpipeline: make: *** No targets specified and no makefile found.  Stop.

I can reproduce them in my xenial machine. So good tests.

**Comment by elopio** [comment]
\o/ and the examples build takes 15 minutes. Enough to end before most reviews.

**Pull request merged**